test/jdk/javax/swing/JScrollBar/4865918/bug4865918.java

```diff
@@ -24,14 +24,16 @@
 /*
  * @test
  * @bug 4865918
- * @requires (os.family != "mac")
+ * @key headful
  * @summary REGRESSION:JCK1.4a-runtime api/javax_swing/interactive/JScrollBarTests.html#JScrollBar
  * @run main bug4865918
  */
 
 import java.awt.Dimension;
 import java.awt.event.MouseAdapter;
 import java.awt.event.MouseEvent;
+import java.awt.Robot;
```

Comment:
```suggestion
import java.awt.Dimension;
import java.awt.Robot;
import java.awt.event.MouseAdapter;
import java.awt.event.MouseEvent;
```

Reply:
ok I see it is being done in other files so modified..